prow/genjobs/cmd/genjobs/main.go

```diff
@@ -125,8 +140,8 @@ func validateOrgRepo(o options, org string, repo string) bool {
 }
 
 // validateJob validates that the job passes validation and should be converted.
-func validateJob(o options, name string, patterns []string) bool {
-	if o.jobBlacklist.Has(name) || (len(o.jobWhitelist) > 0 && !o.jobWhitelist.Has(name)) || !isMatchBranch(o, patterns) {
+func validateJob(o options, name string, patterns []string, jType string) bool {
```

Comment:
I'd be inclined to make this `(o options, name, jType string, patterns ...string)`

Reply:
Is the inclination more about having the `slice` param after the `string` param or about explicitly making the slice variadic. Is this a Golang convention? What is the rationale. The argument being passed is always a `[]string` ATM so I do not see an obvious justification for this to be variadic?